src/stratigraphy/sidebar/a_above_b_sidebar.py

```diff
@@ -45,7 +45,11 @@ def strictly_contains(self, other: AAboveBSidebar) -> bool:
         )
 
     def is_strictly_increasing(self) -> bool:
-        return all(i.value < j.value for i, j in zip(self.entries, self.entries[1:], strict=False))
+        length = len(self.entries)
+        for i in range(length - 1):
+            if self.entries[i].value >= self.entries[i + 1].value:
+                return False
+            return True
```

Comment:
The return statement is on the wrong indentation level. Like this, after checking the first two values, we immediately return, and we never check any later values.

Actually, we can still use all(), even when we want to avoid the zip call:
```suggestion
        return all(
          self.entries[i].value < self.entries[i + 1].value
          for i in range(len(self.entries) - 1)
        )
```
I think that's more readable and less prone to mistakes.